content/workshops/openshift33/exercise1.4.md

```diff
@@ -0,0 +1,223 @@
+<!-- :notebook: You will need to click to expand the "advanced options"<br/><br/> -->
+{{< alert warning >}} You will need to click to expand the "advanced options" {{< /alert >}}
+
+The GitHub repository URL is: <a href='#'>https://github.com/YOUR_ACCOUNT/openshift-workshops.git</a><br/><br/>
+The GitHub context-dir is: <b>/dc-metro-map</b><br/><br/>
+</p>
+
+{{% /panel %}}
+{{< /panel_group >}}
+
+The node.js builder template creates a number of resources for you, but what we care about right now is the build configuration because that contains the webhooks.  So to get the URL:
+
+{{< panel_group >}}
+{{% panel "CLI Steps" %}}
+
+<blockquote>
+<i class="fa fa-terminal"></i> Goto the terminal and type the following:
+</blockquote>
+
+```bash
+oc describe bc/dc-metro-map | grep -i webhook
+```
+
+<blockquote>
+Copy the Generic webhook to the clipboard
+</blockquote>
+
+{{% /panel %}}
+{{% panel "Web Console Steps" %}}
+
+<blockquote>
+Hover over "Browse" and then click on "Builds"
+</blockquote>
+
+This is going to show basic details for all build configurations in this project
+<p><img title="OpenShift Build Configuration" src="../images/ose-lab-rollbacks-buildconfigs.png" width="500"/></p>
+
+<blockquote>
+Click the "dc-metro-map" build config
+</blockquote>
+
+You will see the summary of builds using this build config
+<p><img title="OpenShift Build Configuration Summary" src="../images/ose-lab-rollbacks-buildconfigsummary.png" width="500"/></p>
+
+<blockquote>
+Click the "Configuration" tab (next to the active Summary tab)
+</blockquote>
+
+<p><img title="OpenShift Deployment Configuration Summary" src="../images/ose-lab-devman-deployconfigconfig.png" width="500"/></p>
+Now you can see the various configuration details including the Github specific and Generic webhook URLs.
+
+<blockquote>
+Copy the Generic webhook to the clipboard
+</blockquote>
+
+{{% /panel %}}
+{{< /panel_group >}}
+
+<br/>
+
+> Now switch back over to github
+
+{{< panel_group >}}
+{{% panel "GitHub Steps" %}}
+
+Let's put the webhook URL into the repository. At the main page for this repository (the fork), you should see a tab bar with code, pull requests, pulse, graphs, and settings.
+
+<p><img title="OpenShift Gogs - Go Git Service - Settings" src="../images/ose-lab-rollbacks-settings.png" width="400"/></p>
+
+<blockquote>
+Click the "Settings" tab
+</blockquote>
+
+Now you will see a vertical list of settings groups.<br/><br/>
+
+<blockquote>
+Click the "Webhooks & services" item
+</blockquote>
+
+<p><img title="OpenShift Webhooks and Services" src="../images/ose-lab-rollbacks-githubwebhooks.png" width="600"/></p>
+
+<blockquote>
+Click the "Add webhook" button
+</blockquote>
+
+<blockquote>
+Paste in the URL you copied
+</blockquote>
+
+<blockquote>
+Disable SSL verification by clicking the button
+</blockquote>
+
+<!-- :information_source: You can learn how to setup SSH in the secrets lab<br/><br/> -->
+**INFO:**You can learn how to setup SSH in the secrets lab.
```

Comment:
These were corrected. Without a list of all alert options, I was unaware that 'alert info' was available until this evening.